driver/razerkbd_driver.h

```diff
@@ -88,7 +89,7 @@
 #define RAZER_STEALTH_ROW_LEN 0x10
 #define RAZER_STEALTH_ROWS_NUM 6
 
-
+#define RAZER_BLACKWIDOW_ELITE_TRANSACTION_ID 0x1F
```

Comment:
It'd be better if you'd just use 0x1F instead of the macro

Reply:
"Magic" value in 13 places could not be better

Reply:
It's not really a magic value, it has some structure behind it but it isn't important to know. If you really want to use a constant, rename it to `RAZER_TRANSACTION_ID_1F` or something like that as it's probably also used by other devices

Reply:
Ok, on the way